### PR TITLE
[stable/newrelic-infrastructure] Upgrade k8s integration and add new configuration

### PR DESCRIPTION
**Pull request by douglascamata**
Signed-off-by: Douglas Camata <dcamata@newrelic.com>

Nopes.

#### What this PR does / why we need it:

* Upgrades the New Relic integration to the most recent release
* Add new environment variables (from the new release) to the chart

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/newrelic-infrastructure/README.md [hunk 1]
```diff
@@ -29,6 +29,8 @@ This chart will deploy the New Relic Infrastructure agent as a Daemonset.
 | `updateStrategy`                | Strategy for DaemonSet updates (requires Kubernetes >= 1.6)  | `RollingUpdate`            |
 | `serviveAccount.create`         | If true, a service account would be created and assigned to the deployment | true |
 | `serviveAccount.name`           | The service account to assign to the deployment. If `serviveAccount.create` is true then this name will be used when creating the service account | |
+| `etcdTlsSecretName`             | Name of the secret containing the cacert, cert and key used for setting the mTLS config for retrieving metrics from ETCD. | |
+| `etcdTlsSecretNamespace`        | Namespace where the the secret specified in `etcdTlsSecretName` was created. | `default`  |
```
Comment (areina) [inline stable/newrelic-infrastructure/README.md]:
```suggestion
| `etcdTlsSecretNamespace`        | Namespace where the secret specified in `etcdTlsSecretName` was created. | `default`  |
```

File: stable/newrelic-infrastructure/values.yaml [hunk 2]
```diff
@@ -109,3 +109,6 @@ updateStrategy: RollingUpdate
 
 # Custom attributes to be passed to the New Relic agent
 customAttribues: "'{\"clusterName\":\"$(CLUSTER_NAME)\"}'"
+
+# etcdTlsSecretName: newrelic-infra-etcd-tls-secret
```
Comment (areina) [inline stable/newrelic-infrastructure/values.yaml:113]: Why this value is commented and the next one is not?

Comment (douglascamata) [reply stable/newrelic-infrastructure/values.yaml:113]: To be consistent with the parameters of the integration itself. The TLS secret name is always required. The namespace where it lives can be omitted and then the default value of `default` is used.

Comment (areina) [reply stable/newrelic-infrastructure/values.yaml:113]: Makes sense, thank you for the explanation.

**Pull request merged**